main.tf

```diff
@@ -43,6 +43,20 @@ data "azuread_domains" "default" {
   only_default = true
 }
 
+data "azuread_users" "owners" {
+  user_principal_names = var.application_owner_user_principal_names
+  ignore_missing = true
+}
+
+data "azuread_groups" "owner_group" {
+  object_ids = var.application_owner_group_object_ids
+}
+
+data "azuread_group" "owner_groups" {
+  for_each = zipmap(data.azuread_groups.owner_group.object_ids,data.azuread_groups.owner_group.object_ids)
+  object_id = each.value
+}
```

Comment:
Can you help me understand the reason for looking up groups twice here?

Reply:
Good call out.  It was meant to be a 2 step process (find the groups that match ids - azuread_group**s** and then get the members - azuread_group).  However, since we're passing in the objectids anyways, removed the first search and just using the variables